packages/scss/src/components/_input.scss

```diff
@@ -224,6 +232,10 @@
 
 	&.is-error {
 		@include formError();
```

Comment:
formError shouldn't be a mixin but be turned into an @extend as it will render into a more compact css.